ArchitectureHooks.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <binaryninjaapi.h>
+#include "../../scc/RefCountObject.h"
```

Comment:
wat. you got IDE'd with that SCC include.

Reply:
https://i.imgur.com/WNjZ4lv.png